build/BuildSteps.cs

```diff
@@ -282,9 +282,17 @@ public static void TestPreSignedArtifacts()
                 if (unSignedPackages.Count() != 0)
                 {
                     var missingSignature = string.Join($",{Environment.NewLine}", unSignedPackages);
-                    ColoredConsole.Error.WriteLine($"This files are missing valid signatures: {Environment.NewLine}{missingSignature}");
+                    ColoredConsole.Error.WriteLine($"These files are missing valid signatures: {Environment.NewLine}{missingSignature}");
                     throw new Exception($"sigcheck.exe test failed. Following files are unsigned: {Environment.NewLine}{missingSignature}");
                 }
+
+                var delaySigned = GetDelaySignedBinaries(targetDir);
```

Comment:
can move this to the function below and rename it to CheckDelaySignedBinaries to avoid duplication

Reply:
Yeah will move the one above too, as that's the same way